requirements-dev.txt

```diff
@@ -1,5 +1,6 @@
 black>=18.9b0
 flake8
+ipython
```

Comment:
do we need this for visualization test?

Reply:
Yes, because of the following import: https://github.com/HazyResearch/fonduer/blob/bef4217e2a40a37bd8d0f9b7803a4e0d467eadbe/src/fonduer/utils/visualizer.py#L8